.github/actions/test-template-action/action.yml

```diff
@@ -0,0 +1,89 @@
+name: 'Test template'
+description: 'Tests MC Extension template'
+
+inputs:
+  template-name:
+    description: 'Name of the template to test'
```

Comment:
Nit: this is currently used as e.g. `template-starter-typescript` or `template-custom-view-starter-typescript`

When I think about the `create-mc-app` CLI options, the `--template` option is either `starter` or `starter-template`.

I'm wondering if maybe we could have a similar properties here like `application-type` and `template-name`.

Example usage:

```yaml
- name: Test Custom Application typescript starter template
  uses: ./.github/actions/test-template-action
  with:
    application-type: custom-application
    template-name: starter-typescript
```

With these two properties, we can then build the command names or whatever we need to. The main benefit I see with this is naming consistency and familiarity.

Just a suggestion anyway, happy to hear your thoughts 🤗

Reply:
Yes, I agree it's easier to understand.

Updated here: adc437d55708f9611433001be0dcff3bad79f6f8